### PR TITLE
Set options + dismiss for "Finish site setup"

### PR DESCRIPTION
**Pull request by timarney**
- Adds dismiss "Finish site setup" + updated some text.
- Sets "site search" + "breadcrumb" options

Screen Shot 2022-03-18 at 9 19 25 AM

Closes 
https://github.com/cds-snc/gc-articles-issues/issues/269
https://github.com/cds-snc/gc-articles-issues/issues/273

### CODE REVIEW

File: wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php [hunk 1]
```diff
@@ -67,8 +73,9 @@ public function setOptions()
             update_option("collection_mode", "maintenance");
             update_option("blog_public", 0);
             update_option("options_set", 1);
-            update_option("options_set", 1);
             update_option("blogdescription", get_bloginfo("name"));
+            update_option("show_search", "on");
+            update_option("show_breadcrumbs", "on");
```
Comment (pcraig3) [inline wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php:78]: ooh, good one

File: wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php [hunk 2]
```diff
@@ -106,8 +119,9 @@ public function finishPanel()
         } ?>
         <div class="wrap">
             <div class="finish-setup-content">
+                <span id="finish-setup-dismiss" class="notice-dismiss"><span class="screen-reader-text"><?php _e("Dismiss", 'cds'); ?></span></span>
                 <h3><?php _e('Finish Site Setup', 'cds-snc'); ?></h3>
-                <p><?php _e("Your almost done.  Let's create some pages, articles and default settings.", 'cds-snc'); ?></p>
+                <p><?php _e("Your almost done.  Let's create some pages and default settings.", 'cds-snc'); ?></p>
```
Comment (pcraig3) [inline wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php:124]: Could we say "You‘re" (you are) almost done?

File: wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php [hunk 3]
```diff
@@ -106,8 +119,9 @@ public function finishPanel()
         } ?>
         <div class="wrap">
             <div class="finish-setup-content">
+                <span id="finish-setup-dismiss" class="notice-dismiss"><span class="screen-reader-text"><?php _e("Dismiss", 'cds'); ?></span></span>
```
Comment (pcraig3) [inline wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php:122]: The button is kind of low contrast-y, but we are admins so no big deal really.

**Review by pcraig3 — APPROVED** [review]
Looks good!

The only thing I would note (which is potentially fixed by merging this PR now that we have an "x" button) is that it also shows up for the "template-site" itself, which seems weird when you see it.

But yeah, you can just "x" it off, all good.

File: wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php (hunk 3 above)
Comment (timarney) [reply wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php:122]: Ya just using whatever is built in.

Comment (timarney) [reply wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSetup.php:122]: Fixed.

**Pull request merged**